internal/pkg/service/buffer/store/task.go

```diff
@@ -0,0 +1,37 @@
+package store
+
+import (
+	"context"
+
+	etcd "go.etcd.io/etcd/client/v3"
+
+	"github.com/keboola/keboola-as-code/internal/pkg/service/buffer/store/key"
+	"github.com/keboola/keboola-as-code/internal/pkg/service/buffer/store/model"
+	serviceError "github.com/keboola/keboola-as-code/internal/pkg/service/common/errors"
+	"github.com/keboola/keboola-as-code/internal/pkg/service/common/etcdop/op"
+	"github.com/keboola/keboola-as-code/internal/pkg/telemetry"
+)
+
+func (s *Store) GetTask(ctx context.Context, taskKey key.TaskKey) (r model.Task, err error) {
+	_, span := s.tracer.Start(ctx, "keboola.go.buffer.store.GetTask")
+	defer telemetry.EndSpan(span, &err)
+
+	task, err := s.getTaskOp(ctx, taskKey).Do(ctx, s.client)
+	if err != nil {
+		return model.Task{}, err
+	}
+	return task.Value, nil
```

Comment:
@michaljurecko sorry že zase otravuju, ale už zas nevím. 😕 

V E2E testu jsem dal task do initial state, ale tady `GetTask` (volanej ze [service/buffer/api/service/task.go](https://github.com/keboola/keboola-as-code/pull/1190/files#diff-023bff154a567751036384860d08e7cbf695ff1df9ca583c521d41da2df464a9)) failuje na Not Found chybě.

![image](https://user-images.githubusercontent.com/215660/218443315-ea490a80-9070-4f0d-9c82-489f9f5f6843.png)


V actual state testu ho vidím:
![image](https://user-images.githubusercontent.com/215660/218442291-ded2dba9-fa45-427a-98f9-13a1b79d69eb.png)

Ale když tady začnu dělat query přímo do etcd, tak tam nenajde ani ten konkrétní klíč, ani jiný task:
![image](https://user-images.githubusercontent.com/215660/218443081-6eb7beb1-c52b-44cd-adee-0cec551bf118.png)

Nenapadá tě co mám blbě? Klíč jsem zkopíroval správně, to jsem kontroloval několikrát. 😅

Reply:
:eyes:

Reply:
pred testom to tam je, po teste to tam je, len pocas testu to tam nie je :slightly_smiling_face: 

idem si dat obed a pokracujem, to bude zas nejaka sranda

![image](https://user-images.githubusercontent.com/19371734/218450440-763ba05f-1dbc-460a-8544-6efad8efb85a.png)

Reply:
@JakubMatejka `/config` v initial etcd ti zacina s `/` ale `task` nie, skumam dalej.